addons/bestja_page_fixtures_ucw/templates/pages/become_volunteer.xml

```diff
@@ -117,19 +117,19 @@
                                 <div class="col-md-8 col-md-offset-2">
                                     <ul class="like_write">
                                         <li class="mb64">
-                                            <span>
```

Comment:
But I will change links in the footer to Facebook and email address, and they will be pulled from settings.